bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/graces/Graces.scala

```diff
@@ -30,40 +30,47 @@ case class Graces(blueprint: SpGracesBlueprint, exampleTarget: Option[SPTarget])
       this.db = None
     }
 
-  // R = Phase-I target R-band magnitude
+  // R = Phase-I target R-band or V-band magnitude
   val rMag: Option[Double] =
     for {
       t <- exampleTarget
-      m <- t.getTarget.getMagnitude(Band.R).asScalaOpt
+      m <- t.getTarget.getMagnitude(Band.R).asScalaOpt orElse
```

Comment:
Unrelated to this PR but should we consider if this must follow the RLike band concept? perhaps a question for @andrewwstephens